test/utils.jl

```diff
@@ -38,19 +38,38 @@ end
+        @test Atom.fullREPLpath(raw"Users\ads\.julia\dev\At om\src\repl.jl:25") == (Atom.fullpath(raw".\Users\ads\.julia\dev\At om\src\repl.jl"), 25)
+        @test Atom.fullREPLpath(raw".\Us ers\ads\.julia\dev\At om\src\repl.jl:25") == (Atom.fullpath(raw".\Us ers\ads\.julia\dev\At om\src\repl.jl"), 25)
+        @test Atom.fullREPLpath(raw"..\Us ers\ads\.julia\dev\At om\src\repl.jl:25") == (Atom.fullpath(raw"..\Us ers\ads\.julia\dev\At om\src\repl.jl"), 25)
     else
-        @test Atom.fullREPLpath("@ Atom /home/user/foo/.julia/bar.jl:25") == ("/home/user/foo/.julia/bar.jl", 25)
-        @test Atom.fullREPLpath("/home/user/foo/.julia/bar.jl:25") == ("/home/user/foo/.julia/bar.jl", 25)
-        @test Atom.fullREPLpath("./foo/bar.jl:1") == (Atom.fullpath("./foo/bar.jl"), 1)
-        @test Atom.fullREPLpath("foo/bar.jl:1") == (Atom.fullpath("./foo/bar.jl"), 1)
+        @test Atom.fullREPLpath("@ Atom /home/Users/ads/.julia/dev/Atom/src/repl.jl:25") == ("/home/Users/ads/.julia/dev/Atom/src/repl.jl", 25)
+        @test Atom.fullREPLpath("@ Atom /home/Users/ads/.julia/dev/At om/src/repl.jl:25") == ("/home/Users/ads/.julia/dev/At om/src/repl.jl", 25)
+        @test Atom.fullREPLpath("@ Atom Users/ads/.julia/dev/At om/src/repl.jl:25") == ("Users/ads/.julia/dev/At om/src/repl.jl", 25)
+
+        @test Atom.fullREPLpath("/home/Us ers/ads/.julia/dev/At om/src/re pl.jl:25") == ("/home/Us ers/ads/.julia/dev/At om/src/re pl.jl", 25)
+        @test Atom.fullREPLpath("/home/Us ers/ads/.julia/dev/At om/src/repl:25") == ("/home/Us ers/ads/.julia/dev/At om/src/repl", 25)
+        @test Atom.fullREPLpath("/home/Us ers/ads/.julia/dev/At om/src/repl") == ("/home/Us ers/ads/.julia/dev/At om/src/repl", 0)
+        @test Atom.fullREPLpath("/home/Us ers/ads/.julia/dev/At om/src/repl.jl") == ("/home/Us ers/ads/.julia/dev/At om/src/repl.jl", 0)
+        @test Atom.fullREPLpath("/home/Us ers/ads one/julia/dev/At om/src/repl.jl") == ("/home/Us ers/ads one/julia/dev/At om/src/repl.jl", 0)
+        @test Atom.fullREPLpath("/home/Us ers/ads one/julia/dev/At om/src/re pl.jl") == ("/home/Us ers/ads one/julia/dev/At om/src/re pl.jl", 0)
+
+        @test Atom.fullREPLpath("Users/ads/.julia/dev/Atom/src/repl.jl:25") == (Atom.fullpath("Users/ads/.julia/dev/Atom/src/repl.jl"), 25)
+        @test Atom.fullREPLpath("Users/ads/.julia/dev/At om/src/repl.jl:25") == (Atom.fullpath("Users/ads/.julia/dev/At om/src/repl.jl"), 25)
+        @test Atom.fullREPLpath("Users/ads/.julia/dev/At om/src/repl.jl:25") == (Atom.fullpath("./Users/ads/.julia/dev/At om/src/repl.jl"), 25)
+        @test Atom.fullREPLpath("./Us ers/ads/.julia/dev/At om/src/repl.jl:25") == (Atom.fullpath("./Us ers/ads/.julia/dev/At om/src/repl.jl"), 25)
+        @test Atom.fullREPLpath("../Us ers/ads/.julia/dev/At om/src/repl.jl:25") == (Atom.fullpath("../Us ers/ads/.julia/dev/At om/src/repl.jl"), 25)
```

Comment:
I think the uses of [`Atom.fullpath`](https://github.com/JunoLab/Atom.jl/blob/master/src/utils.jl#L85-L86) here is less useful; in this case it always returns a given string since the path doesn't actually exist.
So maybe just using the original string would be better.
Not sure why we used `fullpath` here before, but I would like to not use it here and rather make a separate test for that later.